src/teleop/gui/src/components/CommReadout.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="wrap">
+      <span>TX: {{tx}} RX: {{rx}}</span>
+    </div>
+  </template>
+  
+  <script>
+import ROSLIB from 'roslib/src/RosLib';
+
+  export default {
+    data() {
+        return {
+            comm_sub: null,
+            tx: "0.00",
```

Comment:
I think it would make more sense to have these be floats and then have the .toFixed() on line 3 where they are actually rendered in the template